AutoCollection/CorrelationContextManager.ts

```diff
@@ -35,16 +36,16 @@ export class CorrelationContextManager {
     /**
      *  A helper to generate objects conforming to the CorrelationContext interface
      */
-    public static generateContextObject(parentId?: string, operationName?: string, operationId?: string): CorrelationContext {
-        operationId = operationId || Util.newGuid();
+    public static generateContextObject(operationId: string, parentId?: string, operationName?: string, correlationContextHeader?: string): CorrelationContext {
         parentId = parentId || operationId;
         
         if (this.enabled) {
             return {
                 operation: {
                     name: operationName,
                     id: operationId,
-                    parentId: parentId
+                    parentId: parentId,
+                    correlationContextHeader: correlationContextHeader
```

Comment:
Why is this necessary here? If possible, I would like to keep this operation object free of properties not present on https://github.com/Microsoft/ApplicationInsights-dotnet/blob/de488729c28b721a758e0024ea2d4f79909ff797/src/Core/Managed/Shared/Extensibility/Implementation/OperationContext.cs so that the API surface is somewhat consistent